### PR TITLE
List missing plugins

### PR DESCRIPTION
**Pull request by konhi**
List all these awesome plugins in readme, so everyone can know what cool features are built-in.

### CODE REVIEW

**Comment by Araxeus** [comment]
Finally someone takes up the initiative, thanks :)

Could maybe also add some details on
* "in-app-menu" - dark menu + scrollbar
* "playback-speed" - adds a slider that control song speed (kinda obvious ngl 😅)
* "precise-volume" - allow controlling the volume in a way more precisely using mousewheel (customizable volume steps)

and maybe also slightly change the description of "shortcuts" (can set up custom global hotkeys)

**Comment by konhi** [comment]
Pretty thanks for these lovely suggestions! 💕
Could you please specify what words would describe `shortcuts` best? I'm not entirely sure what this plugin does (which confirms that it's good idea to describe it! 😅).

Also, I added few images!

**In-app menu**
image
image

**Playback speed**
image

**Notifications**
image

**Downloader**
image

**Auto confirm when paused**
image

**Comment by Araxeus** [comment]
> Pretty thanks for these lovely suggestions! 💕
> Could you please specify what words would describe `shortcuts` best? I'm not entirely sure what this plugin does (which confirms that it's good idea to describe it! 😅).

I would just add "can setup custom global hotkeys for play/pause/next/previous song"
(I actually still have an open PR with a UI for selecting keybinds&more from a few months ago that wasn't merged yet)

> **In-app menu**
> image

the tray menu is actually not related to that plugin 😋

**Comment by cpiber** [comment]
I believe you are still missing `disable-autoplay`. And I believe it would be helpful to sort the list alphabetically

**Comment by Araxeus** [comment]
> I believe you are still missing `disable-autoplay`. And I think it would be helpful to sort the list alphabetically

- makes every song start in "paused" mode

File: readme.md [hunk 1]
```diff
@@ -36,14 +36,21 @@ Install the `youtube-music-bin` package from the AUR. For AUR installation instr
...
-- **Notifications**: display a notification when a song starts playing
+- **Notifications**: display a [notification](https://user-images.githubusercontent.com/78568641/114102651-63ce0e00-98d0-11eb-9dfe-c5a02bb54f9c.png) when a song starts playing
 - **Touchbar**: custom TouchBar layout for macOS
+- [**Discord**](https://discord.com/): show your friends what you listen to with [Rich Presence](https://user-images.githubusercontent.com/28219076/104362104-a7a0b980-5513-11eb-9744-bb89eabe0016.png)
+- **In-app menu**: allows you to [control app from your tray](https://user-images.githubusercontent.com/61631665/129975597-65491d42-47bd-426a-a1a0-7c91114255ff.png) and [gives bars a fancy, dark look](https://user-images.githubusercontent.com/78568641/112215894-923dbf00-8c29-11eb-95c3-3ce15db27eca.png)
+- [**Last.fm**](https://www.last.fm/): scrobbles support
+- **Playback speed**: listen fast, listen slow! [Adds a slider that controls song speed](https://user-images.githubusercontent.com/61631665/129976003-e55db5ba-bf42-448c-a059-26a009775e68.png)
+- **Precise volume**: customizable volume steps for more comfort, allows controlling the volume in precisely using mousewheel
```
Comment (Araxeus) [inline readme.md]: typo in precise volume description, should be "allows controlling the volume precisely using mousewheel"
(without *in*)

**Comment by konhi** [comment]
- > I would just add "can setup custom global hotkeys for play/pause/next/previous song"
- > typo in precise volume description
- > makes every song start in "paused" mode
- > And I think it would be helpful to sort the list alphabetically
- > the tray menu is actually not related to that plugin 😋

Done! ✔
Thanks for this awesome support.

**Review by th-ch — APPROVED** [review]
Looks good, thanks a lot to both of you for all these improvements to the readme!

**Pull request merged**